### DISCUSSION TITLE
Use Native Camera for Panorama Capture

### DISCUSSION THREAD

**Discussion by Ochasovskiy** [yasirkula/UnityNativeCamera#122] [Q&A]
Is there a way to use Native Camera's panorama functionality to capture panorama images?
I'm looking for some solution to make 360 photos within Unity or at least panorama using native camera app's functionality

**Comment by yasirkula** [comment]
There isn't :-) Though I've a plugin for capturing 360 photos from your Unity scenes: https://github.com/yasirkula/Unity360ScreenshotCapture

**Reply by Ochasovskiy** [reply]
yeah, thanks, but it's not what I was looking for. I need to make 360 or at least panorama from camera feed. Using Unity's ArFoundation or any other native solutions to get then something like cube map. Os what I'm looking for is not get 360 screenshot from unity scene, but get 360 texture/cubemap/panorama/whatever using real camera and show that within unity as a skybox or any other relative way with sphere or cube.

**Reply by yasirkula** [reply]
I hope you find a good solution.
